src/app/drive/views/DriveView/DriveView.tsx

```diff
@@ -144,7 +144,31 @@ const DriveView = (props: DriveViewProps) => {
     try {
       const fileMeta = await fileService.getFile(fileUUID, workspacesToken);
       dispatch(uiActions.setIsFileViewerOpen(true));
-      dispatch(uiActions.setFileViewerItem(fileMeta));
+      dispatch(
+        uiActions.setFileViewerItem({
```

Comment:
Please leave a comment explaining why you had to add the object that way so that when we go to fix it, we know exactly where to look to solve it, please :)